advanced/Scripts/chronometer.sh

```diff
@@ -297,98 +359,155 @@ get_ftl_stats() {
   dns_queries_today_raw="${stats_raw[3]}"
   ads_blocked_today_raw="${stats_raw[5]}"
   ads_percentage_today_raw="${stats_raw[7]}"
-
+  queries_forwarded_raw="${stats_raw[11]}"
+  queries_cached_raw="${stats_raw[13]}"
+  
   # Only retrieve these stats when not called from jsonFunc
   if [[ -z "$1" ]]; then
-    local recent_blocked_raw
     local top_ad_raw
     local top_domain_raw
     local top_client_raw
-  
-    domains_being_blocked=$(printf "%'.0f\n" "${domains_being_blocked_raw}")
-    dns_queries_today=$(printf "%'.0f\n" "${dns_queries_today_raw}")
-    ads_blocked_today=$(printf "%'.0f\n" "${ads_blocked_today_raw}")
-    ads_percentage_today=$(printf "%'.0f\n" "${ads_percentage_today_raw}")
     
-    recent_blocked_raw=$(pihole-FTL recentBlocked)
+    domains_being_blocked=$(printf "%.0f\n" "${domains_being_blocked_raw}")
+    dns_queries_today=$(printf "%.0f\n" "${dns_queries_today_raw}")
+    ads_blocked_today=$(printf "%.0f\n" "${ads_blocked_today_raw}")
+    ads_percentage_today=$(printf "%'.0f\n" "${ads_percentage_today_raw}")
+    queries_cached_percentage=$(printf "%.0f\n" "$(calcFunc "$queries_cached_raw * 100 / ( $queries_forwarded_raw + $queries_cached_raw )")")
+    recent_blocked=$(pihole-FTL recentBlocked)
     top_ad_raw=($(pihole-FTL "top-ads (1)"))
     top_domain_raw=($(pihole-FTL "top-domains (1)"))
     top_client_raw=($(pihole-FTL "top-clients (1)"))
     
-    # Limit strings to 40 characters to prevent overflow
-    recent_blocked="${recent_blocked_raw:0:40}"
-    top_ad="${top_ad_raw[2]:0:40}"
-    top_domain="${top_domain_raw[2]:0:40}"
-    [[ "${top_client_raw[3]}" ]] && top_client="${top_client_raw[3]:0:40}" || top_client="${top_client_raw[2]:0:40}"
+    top_ad="${top_ad_raw[2]}"
+    top_domain="${top_domain_raw[2]}"
+    if [[ "${top_client_raw[3]}" ]]; then
+      top_client="${top_client_raw[3]}"
+    else
+      top_client="${top_client_raw[2]}"
+    fi
   fi
 }
 
+get_strings() {
+  # Expand or contract strings depending on screen size
+  if [[ "$chrono_type" == "large" ]]; then
+    phc_str="         ${COL_DARK_GRAY}Pi-hole"
+    lte_str="       ${COL_DARK_GRAY}AdminLTE"
```

Comment:
`Web` can be used on smaller screens, but `Web Interface` is too long and looks out of place on larger screens. I'm thinking `Web Admin` perhaps?